### PR TITLE
Link To Footer Done

### PR DESCRIPTION
**Pull request by akankshajai**
## Description
Changed titles to links in the footer section 

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: components/footer.js [hunk 1]
```diff
@@ -1,24 +1,34 @@
+import Link from 'next/link'
+
 const footer = () => (
     <footer className="footer">
         <div className="columns">
             <div className="column">
                 <div className="title is-4 red has-text-centered">
-                    <h1>Join</h1>
+                    <Link href="/join">
+                        <h1>Join</h1>
```
Comment (SchaeStewart) [inline components/footer.js]: Replace the h1 with an `a` tag with a className of title. EX:
`<h1>Join</h1>` should become `<a className="title">Join</a>`

File: components/footer.js [hunk 2]
```diff
@@ -1,24 +1,34 @@
...
+                    </Link>
                 </div>
             </div>
             <div className="column">
                 <div className="title is-4 red has-text-centered">
-                    <h1>Code of Conduct</h1>
+                    <Link href="/conduct">
+                        <h1>Code of Conduct</h1>
```
Comment (SchaeStewart) [inline components/footer.js]: Same as above

File: components/footer.js [hunk 3]
```diff
@@ -1,24 +1,34 @@
...
+                    </Link>
                 </div>
             </div>
             <div className="column">
                 <div className="title is-4 red has-text-centered">
-                    <h1>Resources</h1>
+                    <Link href="/resources">
+                        <h1>Resources</h1>
```
Comment (SchaeStewart) [inline components/footer.js]: Same as above

File: components/footer.js [hunk 4]
```diff
@@ -1,24 +1,34 @@
...
+                    </Link>
                 </div>
             </div>
             <div className="column">
                 <div className="title is-4 red has-text-centered">
-                    <h1>Contact Us</h1>
+                    <Link href="/contact">
+                        <h1>Contact Us</h1>
```
Comment (SchaeStewart) [inline components/footer.js]: Same as above

**Review by SchaeStewart — CHANGES_REQUESTED** [review]
Thank you so much for contributing! We really appreciate what you did.

I'm asking you to make a couple of small changes before I merge the PR. The reason for this is when you hover the mouse over the current footer links, it shows a vertical cursor instead of a finger icon. (To see what I'm talking about hover you moues over a link in the nav bar, then hover it over one of the links in the footer).

I've left some comments in the review explaining what to do.  
If you don't want to or don't have time to make these changes, that's totally fine. Just let me know. Also, if you have any additional questions please ask.

Once again, thank you for your contribution.

**Comment by akankshajai** [comment]
I have pushed changes. You can review it @akankshajai
Changed Header To Link

**Pull request merged**